capstone-sys/build.rs

```diff
@@ -40,6 +40,15 @@ extern crate bindgen;
 
 extern crate cc;
 
+#[cfg(feature = "use_bindgen")]
+extern crate regex;
```

Comment:
I should upgrade capstone-sys to the 2018 edition so that you don't need to do this